src/annotator/integrations/html.ts

```diff
@@ -65,7 +106,13 @@ export class HTMLIntegration extends TinyEmitter implements Integration {
     this._htmlMeta = new HTMLMetadata();
     this._prevURI = this._htmlMeta.uri();
 
-    this._sideBySideEnabled = this.features.flagEnabled('html_side_by_side');
+    this._sideBySideOptions = {
+      mode: initialSideBySideMode(
+        this.features.flagEnabled('html_side_by_side'),
+        sideBySideOptions.mode
+      ),
+      initiallyProvidedMode: sideBySideOptions.mode,
```

Comment:
This `initiallyProvidedMode` prop is needed to track what was the explicitly provided value for `mode`, which can be `'auto'`, `'manual'` or `undefined`.

We currently just need to know if `'manual'` was explicitly provided, so that we do not enable side-by-side even if the feature flag changes from `false` to `true` later.